dsc/examples/groups.dsc.yaml

```diff
@@ -0,0 +1,38 @@
+# Example for grouping and groups in groups
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+resources:
+- name: Last Group
+  type: DSC/Group
+  properties:
+    $schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+    resources:
+    - name: Last
+      type: Test/Echo
+      properties:
+        output: Last
+  dependsOn:
```

Comment:
seems like too short ident for `dependsOn`.

Reply:
This `dependsOn` is for the `Last Group` whereas the lines above are the resources within the group, so the indent here is correct